crypto/sleep.c

```diff
@@ -36,7 +37,11 @@ void OSSL_sleep(uint64_t millis)
     usleep(millis * 1000);
 #  endif
 # else
-    usleep(millis * 1000);
+    unsigned int s = (unsigned int)(millis / 1000);
+    useconds_t us = (useconds_t)(millis % 1000);
```

Comment:
DJGPP apparently does not know about useconds_t. Please just use unsigned int here as well.

Reply:
Argh...